content/en/docs/tasks/tools/install-kubectl-macos.md

```diff
@@ -12,8 +12,7 @@ card:
 
 ## {{% heading "prerequisites" %}}
 
-You must use a kubectl version that is within one minor version difference of your cluster.
-For example, a v1.2 client should work with v1.1, v1.2, and v1.3 master.
+You must use a kubectl version that is within one minor version difference of your cluster. For example, a v{{< skew latestVersion >}} client can communicate with v{{< skew prevMinorVersion >}}, and v{{< skew nextMinorVersion >}} control planes.
```

Comment:
@Kartik494 I think when @sftim meant mentioning the version numbers.

Reply:
> version numbers

I did, but that's an optional extra improvement only. The change from “master” to “control plane” is the essential detail.